scripts/roster_reminder.php

```diff
@@ -161,7 +161,7 @@
 		require_once JETHRO_ROOT.'/include/sms_sender.class.php';
 		$notification_sms = '';
 		if (!empty($assignees))  {
-			$sendResponse = SMS_Sender::sendMessage($sms_message, $assignees, FALSE);
+			$sendResponse = SMS_Sender::sendMessage($sms_message, $assignees, TRUE);
```

Comment:
Why do we want the roster reminder to be saved as a person_comm ?

Reply:
Hi Tom! Sorry to have taken so long to get back to you on this. I figured that a roster reminder was a communication to a person, and so should be saved to the communication log. Perhaps it would be better to make this optional?

Reply:
Just threw together an option which would make it optional! :-)